tc/sgx/common/crypto/crypto_utils.cpp

```diff
@@ -36,6 +37,9 @@
 #if _UNTRUSTED_
 #include <openssl/crypto.h>
 #include <stdio.h>
+
+// memcpy_s definition is not present in std C library, hence mapping to memcpy
+#define memcpy_s(dest, dest_size, src, count) memcpy(dest, src, count)
```

Comment:
Can we use memcpy_s definition from https://github.com/hyperledger/avalon/blob/master/tc/sgx/common/c11_support.cpp

Reply:
I'll merge this PR for now keeping above temporary fix as an open issue . This is to unblock dependent PRs